### DISCUSSION TITLE
Tornado with Alveo

### DISCUSSION THREAD

**Discussion by ML-Passionate** [beehive-lab/TornadoVM#137] [General]
Is it possible to use TornadoVM with the new Xilinx Alveo SN1000 FPGA-powered 100GbE SmartNIC?

**Comment by stratika** [comment]
According to the [documentation](https://www.xilinx.com/products/boards-and-kits/alveo/sn1000.html#documentation) and particularly this [video](https://www.youtube.com/watch?v=_kYWk29-s5I&t=83s), this card supports Vitis HLS. So, I believe that TornadoVM would be able to generate OpenCL kernels that can be offloaded on the FPGA device. However, it is recommended a closer look about the version of Vitis.

Currently, TornadoVM has been tested on the Xilinx Alveo U50 card with Vitis 2020.2 version.

**Reply by ML-Passionate** [reply]
Thank you. I will check.
